stl/src/stdhndlr.cpp

```diff
@@ -7,7 +7,7 @@
 
 using new_hand = int(__cdecl*)(size_t);
 
-extern "C" new_hand __cdecl _set_new_handler(new_hand);
+extern "C" new_hand __cdecl _set_new_handler(new_hand) noexcept;

```

Comment:
This is a UCRT function; I have a separate cleanup (PR forthcoming) to simply include `<new.h>` to get this. I recommend reverting this change.